xlive/H2MOD/GUI/imgui_integration/AdvancedSettings.cpp

```diff
@@ -1,21 +1,23 @@
 #include "stdafx.h"
 
 #include "Blam/Engine/game/cheats.h"
-#include "Blam\Engine\Networking\NetworkMessageTypeCollection.h"
-#include "H2MOD\Modules\CustomMenu\CustomMenu.h"
-#include "H2MOD\Modules\CustomMenu\CustomLanguage.h"
-#include "H2MOD\Modules\GamePhysics\Patches\MeleeFix.h"
-#include "H2MOD\Modules\HudElements\HudElements.h"
-#include "H2MOD\Modules\Input\Mouseinput.h"
-#include "H2MOD\Modules\RenderHooks\RenderHooks.h"
-#include "H2MOD\Modules\Shell\Config.h"
-#include "H2MOD\Modules\SpecialEvents\SpecialEvents.h"
-#include "Util\Hooks\Hook.h"
+#include "Blam/Engine/Networking/NetworkMessageTypeCollection.h"
+
+#include "H2MOD/Modules/CustomMenu/CustomMenu.h"
```

Comment:
if you want open a pull to update the includes once and for all, but not for the vendor or 3rd-party libraries

Reply:
I'll open a PR to update them once Kantanomo's json config PR is merged